packages/picasso.js/src/core/chart-components/ref-line/refline.js

```diff
@@ -257,6 +267,34 @@ const refLineComponent = {
       let show = p.show === true || typeof p.show === 'undefined';
 
       if (show) {
+        // Create slope line with labels
+        if (p.slope && p.slope.value !== 0) {
```

Comment:
Line 243 already checks if the value is non-zero. Is the check required here as well?

Reply:
The check at line 253 checks for filtering out the lines to be drawn and the ones that are oob, but this check here is to know if we have to create a ref line or a slope line ....